test/test_thread_safe_queue.cc

```diff
@@ -1,131 +0,0 @@
-/*******************************************************************************
- * Copyright 2019-2020 Intel Corporation
- *
```

Comment:
thread_safe_queue has been moved to a new location but not removed, shouldn't the tests still apply?

Reply:
Yes, I moved it to `examples/cpp` since `infer_multiple_networks` was the only consumer of it. I'm not sure if we need either that utility or another implementation of a thread-safe queue, but I've added it back anyway.